### PR TITLE
[FCE-241] Merge iOS sdk to monorepo

### PR DESCRIPTION
**Pull request by karkakol**
## Description

Merge iOS client library to react-native-client repo

## Motivation and Context

This will make development much easier and fast. No need to create temp release's. All code in single codebase.

### CODE REVIEW

File: packages/ios-client/.gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "protos"]
```
Comment (graszka22) [inline packages/ios-client/.gitmodules:1]: remove?

File: packages/ios-client/scripts/init.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+brew install swift-format
```
Comment (graszka22) [inline packages/ios-client/scripts/init.sh:2]: do we need it? we have init.sh in the root folder

File: packages/ios-client/package.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (graszka22) [inline packages/ios-client/package.json:1]: FYI: if we are no longer using release-it, you can remove it and yarn.lock too

**Review by graszka22 — COMMENTED** [review]
Looks good 👍 I've run this branch on my computer and it built without problems, also I can easily edit files in xcode, nice :3 
Please also add a task about removing ios demo app. And linting just like on android, it would be nice to lint  🪄  everything 🪄  with just `yarn lint`. And releasing process too, but I guess you'll do that with android and rn clients at once.

**Comment by graszka22** [comment]
Also do we still release a swift-package? If not, we can remove related files

File: examples/video-chat/ios/Podfile.lock [hunk 4]
```diff
@@ -1425,6 +1419,6 @@ SPEC CHECKSUMS:
   Yoga: 64cd2a583ead952b0315d5135bf39e053ae9be70
   ZXingObjC: 8898711ab495761b2dbbdec76d90164a6d7e14c5
 
-PODFILE CHECKSUM: 0376e85da25bcfda7c0095d7c9d174ee924ff49d
+PODFILE CHECKSUM: 2c30f3251567073d449ed093b86832597a8e3058
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```
Comment (mironiasty) [inline examples/video-chat/ios/Podfile.lock:1424]: I've added task to lock this version in our repo at some point in future [FCE-253]

File: packages/ios-client/yarn.lock
Comment (mironiasty) [inline packages/ios-client/yarn.lock:1]: This file should be removed

File: packages/ios-client/pre-commit
Comment (mironiasty) [inline packages/ios-client/pre-commit:1]: Is this file still required/used anywhere?

File: packages/ios-client/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+2. Edit `Debug.xcconfig` to set backend url in development.
+3. Run `release-it` to release. Follow the prompts, it should update version in podspec, make a commit and tag and push
+   the new version.
+
+## Contributing
+
+We welcome contributions to iOS Client SDK. Please report any bugs or issues you find or feel free to make a pull
+request with your own bug fixes and/or features.
+
+## Fishjam Ecosystem
```
Comment (mironiasty) [inline packages/ios-client/README.md:41]: This section could be removed

File: packages/ios-client/README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+# Fishjam iOS Client
+
+[Fishjam](https://github.com/fishjam-dev/fishjam) Client library for iOS apps written in Swift.
```
Comment (mironiasty) [inline packages/ios-client/README.md:3]: nit: link can be updated to Fishjam Cloud (but we can also cleanup readme's in other PR)

Comment (karkakol) [reply packages/ios-client/README.md:3]: i suggest another PR

**Comment by karkakol** [comment]
> Also do we still release a swift-package? If not, we can remove related files

We do. Lets allow ios client app exists.

File: packages/ios-client/protos
Comment (graszka22) [inline packages/ios-client/protos:1]: remove?

**Comment by mironiasty** [comment]
> > Also do we still release a swift-package? If not, we can remove related files
> 
> We do. Lets allow ios client app exists.

I don't think we are still using this package 🤔 Does example app use package or link files directly?

**Review by mironiasty — APPROVED** [review]
There are still some cleanups required, but overall it looks good.

**Review by mironiasty — APPROVED** [review]

**Comment by mironiasty** [comment]
@karkakol Before you merge, can you update PR description?

**Review by graszka22 — APPROVED** [review]
